starport/pkg/goenv/goenv.go

```diff
@@ -0,0 +1,21 @@
+// Package goenv defines env variables known by Go and some utilities around it.
+package goenv
+
+import (
+	"go/build"
+	"os"
+	"path/filepath"
+)
+
+const (
+	// GOBIN is the env var for GOBIN.
+	GOBIN = "GOBIN"
+)
+
+// GetGOBIN returns the path of where Go binaries are installed.
+func GetGOBIN() string {
+	if binPath := os.Getenv(GOBIN); binPath != "" {
```

Comment:
Is GOBIN always an absolute path? Otherwise we should use `ExpandEnv` as well

Reply:
Yes, `GOBIN` always is the real path where binaries are located under.